eng/versioning/version_client.txt

```diff
@@ -50,19 +50,19 @@ com.azure:azure-communication-common;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-communication-sms;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-communication-identity;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-communication-phonenumbers;1.0.0-beta.5;1.0.0-beta.5
-com.azure:azure-core;1.13.0;1.14.0-beta.1
+com.azure:azure-core;1.13.0;1.14.0
 com.azure:azure-core-amqp;2.0.2;2.1.0-beta.1
 com.azure:azure-core-amqp-experimental;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-core-experimental;1.0.0-beta.10;1.0.0-beta.11
 com.azure:azure-core-http-jdk-httpclient;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-core-http-netty;1.8.0;1.9.0-beta.1
-com.azure:azure-core-http-okhttp;1.5.0;1.6.0-beta.1
-com.azure:azure-core-management;1.1.1;1.2.0-beta.1
+com.azure:azure-core-http-netty;1.8.0;1.9.0-b
```

Comment:
Is this something new ?
Like new way to ship out a beta release ?
Also, since `azure-cosmos` depends on it, are we going to take dependency on beta version of `azure-core-http-netty` ?